isabelle/Doc/Specification/Core.thy

```diff
@@ -1,287 +0,0 @@
-(*<*)
-theory Core
-  imports
-      Core.Semantics
-      Core.SemanticsTypes
-      "Util.SpecificationLatexSugar"
-
-begin
```

Comment:
Contents of this file were either discarded (as better comments were added) or included in the literal semantics